hw/i386/virt/virt.c

```diff
@@ -185,6 +189,11 @@ static void virt_machine_state_init(MachineState *machine)
     VirtMachineState *vms = VIRT_MACHINE(machine);
     bool linux_boot = (machine->kernel_filename != NULL);
 
+    //TODO: This needs some property to specify
+    //Can be put into VirtMachineState
+    int segment_total_nr = 2;
```

Comment:
I am still trying to see if we should have the ability to add as many segments as we need on the command line vs setting it to a predetermined number of segments. Allowing the user to specify the number of segments will make this looks more like a pcie_pxb along the lines of 

```
-device pxb-pcie,id=pcie.1,bus_nr=x[,numa_node=y][,addr=z]
```